vaadin-popover-flow-parent/vaadin-popover-flow/src/main/java/com/vaadin/flow/component/popover/Popover.java

```diff
@@ -149,6 +148,53 @@ public Registration addOpenedChangeListener(
         return addListener(OpenedChangeEvent.class, listener);
     }
 
+    /**
+     * Sets whether component should open modal or modeless popover. When the
+     * popover is modal, interacting with elements behind it will be prevented
+     * until the popover is closed.
```

Comment:
Would make sense to clarify this to set the correct expectations. Otherwise "interacting with elements behind it will be prevented" could be interpreted as server-side state can not be modified.

Reply:
Thanks, added a note about not using server-side modality.